pkg/agent/sidecar/usecase/initcontainer/initcontainer.go

```diff
@@ -252,5 +268,8 @@ func (r *run) Stop(ctx context.Context) error {
 }
 
 func (r *run) PostStop(ctx context.Context) error {
+	if r.rs != nil {
+		r.rs.PostStop(ctx)
```

Comment:
should be in PreStop.

Reply:
Thank you.:bow:
Got it!
I fixed: https://github.com/vdaas/vald/pull/519/commits/a7f2c670c4bec8c20342972b37d4757089c3fa52